packages/release/logdog.common.conf

```diff
@@ -17,4 +17,4 @@ exec signpost signpost status
 exec wicked wicked show all
 file os-release /etc/os-release
 glob /var/log/kdump/*
-settings settings.json
+settings settings.json
```

Comment:
Also, your editor is not saving a single newline at the end of the file - can you update your config?